experiment/run_experiment.py

```diff
@@ -235,9 +237,13 @@ def start_experiment(  # pylint: disable=too-many-arguments
     config['oss_fuzz_corpus'] = oss_fuzz_corpus
     config['description'] = description
     config['concurrent_builds'] = concurrent_builds
+    config['measurers_cpus'] = measurers_cpus
+    config['runners_cpus'] = runners_cpus
     config['runner_machine_type'] = config.get('runner_machine_type',
                                                'n1-standard-1')
     config['runner_num_cpu_cores'] = config.get('runner_num_cpu_cores', 1)
+    assert (runners_cpus is None or
+            runners_cpus >= config['runner_num_cpu_cores'])
```

Comment:
Why enforce this?

Reply:
well, if i say that runners must run on 2 cores but then each runner needs 3 cores it's not ok.